packages/connect-iframe/README.md

```diff
@@ -1,18 +1,18 @@
 # @trezor/connect-iframe
 
-Build `@trezor/connect` iframe hosted on `connect.trezor.io/*/iframe.html`
+Build `@trezor/connect` `iframe`, which enables secure communication with [@trezor/connect](../connect) for third parties and web version of Suite. Official versions of the `iframe` are hosted on `connect.trezor.io/<version>/iframe.html`
```

Comment:
`@trezor/connect-web` not `@trezor/connect`
iframe is used only in web builds while `@trezor/connect` is nodejs implementantion